.github/workflows/publish.yml

```diff
@@ -38,13 +46,16 @@ jobs:
           key: python-3.6-${{ hashFiles('poetry.lock') }}
           restore-keys: |
             python-3.6-
+
       - name: Setup Environment
-        uses: ./.github/actions/python-poetry-setup
+        # Using non-relative path for correct behavior when used as a remote
+        # workflow.
+        uses: Kitware/SMQTK-Core/.github/actions/python-poetry-setup@master
 
       - name: Publish
+        env:
+          POETRY_PYPI_TOKEN_PYPI: ${{ secrets.PYPI_TOKEN}}
         run: |
           echo "Publishing new tag: ${{ github.ref_name }}"
-
           git checkout ${{ github.ref_name }}
-          poetry config pypi-token.pypi ${{ secrets.PYPI_TOKEN }}
```

Comment:
Just curious, where does `POETRY_PYPI_TOKEN_PYPI` get used? Does the poetry build command automatically search for the token in that environment variable?

Reply:
Yup.